article_json.gemspec

```diff
@@ -5,7 +5,7 @@ Gem::Specification.new do |s|
   s.version = ArticleJSON::VERSION
   s.platform = Gem::Platform::RUBY
 
-  s.authors = %w(@dsager)
+  s.authors = %w(@dsager towanda nicolas-fricke)
```

Comment:
Shouldn't be with the `@`?

Reply:
Good point! In fact `authors` should contain the clear names and no Github handle at all. Changing it :)